examples/Billing/AddBillingSetup.cs

```diff
@@ -212,13 +220,16 @@ FROM billing_setup
                 }
 
                 DateTime lastEndingDateTime = DateTime.Parse(lastEndingDateTimeString);
+
+                // Sets the new billing setup to start one day after the ending date time.
                 billingSetup.StartDateTime = lastEndingDateTime.AddDays(1).ToString("yyyy-MM-dd");
+
+                // Sets the new billing setup to end one day after the starting date time.
                 billingSetup.EndDateTime = lastEndingDateTime.AddDays(2).ToString("yyyy-MM-dd");
             }
             else
             {
-                // If there are no existing approved billing setups, the only acceptable start time
-                // is TimeType.Now.
+                // Otherwise, the only acceptable start time is TimeType.Now.
                 billingSetup.StartTimeType = TimeType.Now;
```

Comment:
Make sure to set billingSetup.EndDateTime in this else block too with `TimeType.Now + 1 day`

Reply:
Fixed.